olive/data/component/text_generation.py

```diff
@@ -65,7 +65,7 @@ class TextGenParams(ConfigBase):
     # if false, cannot gaurantee all sequences are same length. data loader will have to handle this during collation
     pad_to_max_len: bool = True  # pad sequences to max_len, ignored for JOIN corpus strategy
     drop_short_sequences: bool = False  # drop sequences shorter than max_len. Mutually exclusive with pad_to_max_len
-    add_special_tokens: bool = True  # add special tokens, ignored for JOIN corpus strategy
+    add_special_tokens: bool = True  # add bos and eos tokens
```

Comment:
Do we need to split it as:
add_bos_tokens
add_eos_tokens
?

As you said, for llama model, the eos_token_id is missed. But for bos? Should we handle the bos and eos separately?

Reply:
I wasn't sure about whether we should separate the two. since for text-generation, we normally add both tokens.

Reply:
we pass `add_special_tokens=False` when calling the tokenizer. so, there won't be any duplicates. 

I can separate them into bos and eos. Will give us better control over this in the future if needed.